moveit_kinematics/kdl_kinematics_plugin/include/moveit/kdl_kinematics_plugin/chainiksolver_vel_mimic_svd.hpp

```diff
@@ -62,14 +62,7 @@ class ChainIkSolverVelMimicSVD : public ChainIkSolverVel
                                     const std::vector<kdl_kinematics_plugin::JointMimic>& mimic_joints,
                                     bool position_ik = false, double threshold = 0.001);
 
-// TODO: simplify after kinetic support is dropped
-#define KDL_VERSION_LESS(a, b, c) ((KDL_VERSION) < ((a << 16) | (b << 8) | c))
-#if KDL_VERSION_LESS(1, 4, 0)
   void updateInternalDataStructures();
```

Comment:
Shouldn't this just have an override (https://github.com/ros-planning/moveit/blob/master/moveit_kinematics/kdl_kinematics_plugin/include/moveit/kdl_kinematics_plugin/chainiksolver_vel_mimic_svd.hpp#L70)? Not sure why this was changed in the first place.

Reply:
Related to https://github.com/ros-planning/moveit2/pull/236

Reply:
From the git blame, it looks like it was removed here:
https://github.com/ros-planning/moveit2/commit/68c7efc73ec78588a54916bc2f8df93a6831ce3e#diff-3a912bc5f8c8738aafa920927d7dc938

Reply:
Yeah, looks like it slipped through. But then I would opt for reverting to the function override and only remove the version check.

Reply:
Before foxy, we didn't have this function at all [see](https://github.com/ros2/orocos_kinematics_dynamics/blob/eloquent/orocos_kdl/src/chainiksolver.hpp), they add it to the foxy branch because it was missing [see](https://github.com/ros2/orocos_kinematics_dynamics/blob/foxy/orocos_kdl/src/chainiksolver.hpp)

Reply:
Feel free to push commits for what you mean, and I'll run it past CI build.

Reply:
I think it's already included in #227 so if that one runs passes CI on Foxy we should be good.